src/NuGet.Core/NuGet.Packaging/PackagePathResolver.cs

```diff
@@ -25,6 +25,13 @@ public PackagePathResolver(string rootDirectory, bool useSideBySidePaths = true)
                     string.Format(Strings.StringCannotBeNullOrEmpty, nameof(rootDirectory)),
                     nameof(rootDirectory));
             }
+            if (!Path.IsPathRooted(rootDirectory))
```

Comment:
I know it's already missing, but it'd be great if you add some quick docs on the constructor.